### PR TITLE
Ovchinnikov/add db tables

### PR DESCRIPTION
**Pull request by STALINovsky**
Add 2 tables to database: TaskStates and TaskTracks.

### CODE REVIEW

File: DIMS-Core.Database/DIMS-Core.Database/Tables/TaskStates.sql [hunk 1]
```diff
@@ -0,0 +1,5 @@
+﻿CREATE TABLE [dbo].[TaskStates]
+(
+	[StateId] INT NOT NULL PRIMARY KEY IDENTITY,
+	[StateName] NVARCHAR(250) NOT NULL UNIQUE
```
Comment (maximsan) [inline DIMS-Core.Database/DIMS-Core.Database/Tables/TaskStates.sql:4]: I believe 50 is more than enough here

**Review by maximsan — APPROVED** [review]
Looks fine for me. @PushMeNow  what do u think ?

**Review by PushMeNow — APPROVED** [review]
LGFM. Approved

File: DIMS-Core.Database/DIMS-Core.Database/Tables/TaskStates.sql (hunk 1 above)
Comment (STALINovsky) [reply DIMS-Core.Database/DIMS-Core.Database/Tables/TaskStates.sql:4]: @maximsan Ok.I'll make it.

**Review by dbogdan2000 — APPROVED** [review]

**Pull request merged**